src/components/SessionsGrid.jsx

```diff
@@ -190,11 +190,11 @@ const StyledDiv = styled.div`
 function SessionsGrid({ sessions }) {
   const pastSessions = sessions
     .filter((session) => {
-      return session.date < addTwoHours(new Date());
+      return addTwoHours(session.date) < new Date();
```

Comment:
Good idea :)

Reply:
Yeah, I want the session of today to be visible as upcoming session at the start of the session.